terraform/aws/minikube/k8s-userdata.tpl.sh

```diff
@@ -2,16 +2,17 @@
 echo "nameserver 8.8.8.8" >> /etc/resolv.conf
 
 # Install AWS utils
-yum install -y wget unzip python3-pip.noarch
+apt-get update
+apt-get install -y wget unzip python3-pip
 pip3 install awscli
 export PATH=\$$PATH:/usr/local/bin
 
 # Copy kubeconfig to S3 bucket
-aws s3 cp /home/centos/kubeconfig s3://${cluster_name}/kubeconfig_${cluster_name}
+aws s3 cp /home/ubuntu/kubeconfig s3://${cluster_name}/kubeconfig_${cluster_name}
```

Comment:
Maybe good idea will be use `$HOME`?

Like this: `"$HOME/kubeconfig"`

Reply:
keep to next time this change